src/condor_tools/htcondor_cli/annex_validate.py

```diff
@@ -43,18 +47,26 @@ def __str__(self):
         rv += f", host_name='{self.host_name}'"
         rv += f", default_queue='{self.default_queue}'"
         rv += f", batch_system='{self.batch_system}'"
-        rv += f", script_base='{self.script_base}'"
+        rv += f", executable='{self.executable}'"
+        other_scripts_list = ", ".join(other_scripts)
+        rv += f", other_scripts='{other_scripts_list}'"
         rv += f", allocation_required={self.allocation_required}"
         rv += ")"
         return rv
 
 
-    def get_constraints(self, queue_name, gpus, gpu_type):
+    def get_constraints(self, queue_name: str, gpus: Optional[str], gpu_type: Optional[str]) -> Optional[dict]:
```

Comment:
```suggestion
    def get_constraints(self, queue_name: str, gpus: Optional[int], gpu_type: Optional[str]) -> Optional[dict]:
```
based on how I see it used in Bridges2System

Reply:
Brain fart.  Good catch.